### ISSUE TITLE
Website is not showing properly in GitHub

### ISSUE THREAD

**Issue by hazeljpw** [hazeljpw/Sanford-Sweets#2]
After deployment in GitHub, the website don't seem to show CSS and Bootstrap properly. It looks like just HTML with background.

**Comment by hazeljpw** [comment]
Referred to Bootstrap course forum to find solution and found out that it may have something to do with node_modules. After reviewing my codes, I notice that my .gitignore is not removing node_modules properly, probably because I have it pushed to this repo before putting in .gitignore. I fixed the this error by putting this in Vs Code Terminal:

git rm -rf --cached .
git add .

After that, I add, commit and push the changes.

**Comment by hazeljpw** [comment]
The issue is still not getting resolve, I tried to host the page in Netlify and it seems to be working except some bootstrap and font-awesome issues. 

I reviewed my code again and I found an issue that my CSS source actually have node_modules so it's taking in from node_modules files. I refer to documentations and a new link for Font Awesome and Bootstrap. 

The website deployed with GitHub is now showing Bootstrap and Font Awesome Icons.
